### PR TITLE
Fix typos in ActionCable Channel [ci skip]

### PR DESCRIPTION
**Pull request by tomkadwill**
### Summary

Documentation fixes for action cable channel.

### CODE REVIEW

File: actioncable/lib/action_cable/channel/periodic_timers.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ module PeriodicTimers
       end
 
       module ClassMethods
-        # Allows you to call a private method <tt>every</tt> so often seconds. This periodic timer can be useful
```
Comment (vipulnsward) [inline actioncable/lib/action_cable/channel/periodic_timers.rb]: Let's continue having this here.

Comment (jeremy) [inline actioncable/lib/action_cable/channel/periodic_timers.rb]: Agree. It looks like a mistake here, but it's meant to be read as "periodically call a method every N seconds."

> Sets a timer to periodically call a method. Specify the period in seconds with the <tt>every:</tt> keyword argument, e.g. `every: 30.seconds`. …

File: actioncable/lib/action_cable/channel/periodic_timers.rb [hunk 2]
```diff
@@ -12,8 +12,8 @@ module PeriodicTimers
...
-        # for sending a steady flow of updates to a client based off an object that was configured on subscription.
+        # Allows you to call a private method periodically. Specify the period, in seconds, using the <tt>every</tt> keyword.
```
Comment (vipulnsward) [inline actioncable/lib/action_cable/channel/periodic_timers.rb]: `keyword argument` or `option`

**Comment by vipulnsward** [comment]
Thanks @tomkadwill !

**Pull request merged**